index.html

```diff
@@ -790,12 +790,7 @@ <h3>Presentations</h3>
 
         <p>
 A <a>verifiable presentation</a> expresses data from one or more
-<a>verifiable credentials</a>, and is packaged in such a way that the
-authorship of the data is <a>verifiable</a>. If <a>verifiable credentials</a>
-are presented directly, they become <a>verifiable presentations</a>. Data
-formats derived from <a>verifiable credentials</a> that are cryptographically
-<a>verifiable</a>, but do not of themselves contain
-<a>verifiable credentials</a>, might also be <a>verifiable presentations</a>.
+<a>verifiable credentials</a>.
```

Comment:
0 or more credentials AND arbitrary additional data encoded in JSON-LD

Reply:
Re-worded in [12f46a4](https://github.com/w3c/vc-data-model/pull/1181/commits/12f46a47beeeeb949b9ed15b6553c06338a6ec87)

Reply:
Re-worded in [29f5cc9](https://github.com/w3c/vc-data-model/pull/1181/commits/29f5cc974498b19b1f4062f4ddc05bdeb5e954f5)